config-templates/shopify/order_line.yaml

```diff
@@ -0,0 +1,47 @@
+# ORDER_LINE CONFIG
+data_source:
+  name: shopify_orders_line
+  owners:
+    - support@transformdata.io
+  sql_query: |
+    select * from SHOPIFY.ORDER_LINE a left join SHOPIFY."ORDER" b on a.order_id = b.id
+  mutability:
+    type: immutable
+  identifiers:
+    - name: id
+      type: primary
+    - name: order
+      type: foreign
+      expr: order_id
+    - name: product_id
```

Comment:
same comment as inventory_level - many of these foreign keys don't have matching primary keys. i think it might be confusing and I think we may send validation warnings